src/sentry/static/sentry/app/components/dropdownAutoCompleteMenu.jsx

```diff
@@ -173,17 +175,31 @@ const StyledLabel = styled('div')`
   border-width: 1px 0;
 `;
 
-const StyledMenu = styled(({isOpen, ...props}) => <div {...props} />)`
+const StyledMenu = styled(({isOpen, alignMenu, ...props}) => <div {...props} />)`
   background: #fff;
   border: 1px solid ${p => p.theme.borderLight};
   border-radius: 0 ${p => p.theme.borderRadius} ${p => p.theme.borderRadius}
     ${p => p.theme.borderRadius};
   position: absolute;
   top: calc(100% - 1px);
-  left: 0;
   min-width: 250px;
-  font-size: 0.9em;
   z-index: 1;
+  max-height: 300px;
+  overflow-y: scroll;
+
+  ${({alignMenu, theme: {borderRadius}}) => {
+    if (alignMenu == 'left')
+      return `
+      left: 0;
+      border-radius: 0 ${borderRadius} ${borderRadius} ${borderRadius};
+    `;
+    else if (alignMenu == 'right')
```

Comment:
Is there ever a case where we wouldn't want one of these styles? We should have a default in the case the developer makes a mistake

Reply:
tho i think `left` always supersedes right, so if we set the default to `right` then we might not need it.

Reply:
Yeah just saying we probably only want an if/else instead of if/elseif/else